internal/regopolicyinterpreter/regopolicyinterpreter.go

```diff
@@ -590,11 +590,11 @@ func (r *RegoPolicyInterpreter) Query(rule string, input map[string]interface{})
 		return nil, err
 	}
 
-	result := make(RegoQueryResult)
 	if len(rawResult) == 0 {
-		return result, nil
+		return nil, errors.New("emtpy result from Rego query")
```

Comment:
Is there any way to get context around why the rego query is empty or what to do about it?